internal/configure.go

```diff
@@ -203,14 +203,34 @@ func ConfigureUploader() (uploader *Uploader, err error) {
 	}
 
 	folder := uploader.UploadingFolder
-	deltaFileManager := uploader.deltaFileManager
 
 	compressor, err := configureCompressor()
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to configure compression")
 	}
 
-	uploader = NewUploader(compressor, folder, deltaFileManager)
+	uploader = NewUploader(compressor, folder)
+	return uploader, err
+}
+
+// ConfigureWalUploader connects to storage and creates an uploader. It makes sure
+// that a valid session has started; if invalid, returns AWS error
+// and `<nil>` values.
+func ConfigureWalUploader() (uploader *WalUploader, err error) {
+	uploader, err = ConfigureMockWalUploaderWithoutCompressMethod()
```

Comment:
ConfigureMockWalUploader looks like a function for testing something with mock, but you use it in here

Reply:
This is a typo, thanks for spotting.